sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -296,71 +294,68 @@ public enum CjkBigramTokenFilterScripts
     }
     public partial class ClassicTokenizer : Azure.Search.Documents.Models.Tokenizer
     {
-        public ClassicTokenizer() { }
+        public ClassicTokenizer(string name) : base (default(string)) { }
         public int? MaxTokenLength { get { throw null; } set { } }
     }
     public partial class CognitiveServicesAccount
     {
         public CognitiveServicesAccount() { }
         public string Description { get { throw null; } set { } }
-        public string ODataType { get { throw null; } }
     }
     public partial class CognitiveServicesAccountKey : Azure.Search.Documents.Models.CognitiveServicesAccount
```

Comment:
@heaths - And we should probably make this an `AzureKeyCredential` too

Reply:
#10815